dataland-frontend/src/types/HighImpactClimateSectors.ts

```diff
@@ -10,22 +10,22 @@ export const HighImpactClimateSectorsKeys: { [key: string]: string } = {};
 
 export const optionsForHighImpactClimateSectors = naceCodeTreeFilteredByHighImpactClimateSectors.map((sector) => {
   const key = sector.key as string;
-  const keyDescription = sector.label?.split(" - ")[1].trim() ?? `NACE Code ${key} in GWh`;
-  HighImpactClimateSectorsKeys[`NaceCode${key}InGWh` as keyof typeof HighImpactClimateSectorsKeys] = keyDescription;
+  const keyDescription = sector.label?.split(" - ")[1].trim() ?? `NACE Code ${key}`;
```

Comment:
Ok, i will it open until Julia has responded

Reply:
I talked to Julia and we found that this isse happens also at other stages, especially when asking for uploading documents in LkSG and SFDR. The problem basically is that we use the same tooltip for the upload and the view page. We thus have two alternatives: introduce a second column of descriptions to separate view and upload pages or adjust the existing descriptions to fit both. The latter approach is preferable as it keeps the data dictionaries simple. However, for the moment, we will leave all of them as they are. Julia will discuss this with Machteld and create a JIRA ticket to address all of them in one iteration in case that this is considered desired.

Reply:
ok, then i will close this here